nexus/Freqlog/Definitions.py

```diff
@@ -15,9 +13,10 @@ class Defaults:
         {chr(i) for i in range(ord('a'), ord('z') + 1)} | {chr(i) for i in range(ord('A'), ord('Z') + 1)}
     DEFAULT_ALLOWED_CHARS: set = \
         DEFAULT_ALLOWED_FIRST_CHARS | {"'", "-", "_", "/", "~"}  # | {chr(i) for i in range(ord('0'), ord('9') + 1)}
+    DEFAULT_MODIFIERS: set = [
+            'left_control',  'left_alt',  'left_meta',  'left_super',  'left_hyper',
+            'right_control', 'right_alt', 'right_meta', 'right_super', 'right_hyper']
```

Comment:
Are these string values not defined in vinput? Can they be defined there and use `vinput.<name>` constants here?

Reply:
Shift is one of them, I think it's better to use these like this explicitly. There are no definitions for these, because they are attributes of KeyboardModifier

Reply:
Okay, I'm fine with that.